base/pkg.jl

```diff
@@ -13,9 +13,17 @@ type PkgError <: Exception
     msg::AbstractString
 end
 
-for file in split("dir types reqs cache read query resolve write entry git")
-    include("pkg/$file.jl")
-end
+include("pkg/dir.jl" )
```

Comment:
I think the old way is more concise. Some packages do things this way, and choosetests is similar in spirit.